CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 9.1.0 (2024-05-21)
+
+- Pin `sqlite3` dependency due to upstream issue.
```

Comment:
This is for internal testing only. As this is not a user facing change so it doesn't require a mention in the change log.

```suggestion
```